packages/ui-kit/src/components/Leaderboard/Leaderboard.types.ts

```diff
@@ -67,6 +89,15 @@ export interface LeaderboardProps extends ErrorFallbackProps, React.ComponentPro
   /** Leaderboard query props */
   query?: LeaderboardQueryProps
 
-  /** Format function for labels, must return an array with the new labels */
+  /** Optional props that are used to configure the table component. */
+  tableProps?: LeaderboardTableProps
+
+  /** Optional props that are used to configure the chart component. */
+  chartProps?: LeaderboardChartProps
```

Comment:
This and `chartConfigProps` is only relevant when `variant: 'bar'`?

Reply:
Yes

Reply:
Maybe we should update the comments to say something like

```
  /** Optional props that are used to configure the chart component. Only used when `variant` is "bar". */
```

Reply:
Done!